tests/unit/backend/recipe/algorithm/test_LoadGroupingDefinition.py

```diff
@@ -302,35 +308,35 @@ def test_load_from_xml_file_with_instrument_donor(self):
     def test_load_from_xml_file_with_instrument_file(self):
         self.do_test_load_with_instrument_file("xml")
 
-    # NOTE commented out because slow
-    # def test_load_from_xml_file_with_instrument_name(self):
-    #     self.do_test_load_with_instrument_name("xml")
+    # TODO THIS IS BAD -- EWM 5043
+    @pytest.mark.xfail(reason="Cannot load test instrument from name", strict=True)
+    def test_load_from_xml_file_with_instrument_name(self):
+        self.do_test_load_with_instrument_name("xml")
```

Comment:
This was a newly-discovered error.  It fails because mantid cannot find the test instrument file by its name.  Or, if using "SNAP" as the name, because SNAP has far more pixels than the test instrument.